{{cookiecutter.app_name}}/helpers/database.go

```diff
@@ -0,0 +1,18 @@
+package helpers
+
+import (
+	"errors"
+
+	"github.com/spf13/viper"
+	"gorm.io/gorm"
+)
+
+// Get database connection from viper config
```

Comment:
Why separating this from `bootstrap/database.go`? Couldn't this be together? I don't see the point of having a helper for that 🤔 Also note that I did not see where `GetDBConnection` is actually used. I would assume that `InitDatabase` would use it but it's not. So for now, it seems like an unused method.

Reply:
Gorm does not have any function to get database connection after initialize so I need to create a function to get database connection from viper config. I think `bootstrap` should contains only initialize stuffs. That's why I decide to put `GetDBConnection` function at helpers. But yes this function is unused for now, I will remove this in this PR and will put it again in the PR that need to use database connection.

Removed in 28bb42380572dbecf71a6340c578da3378707553

Reply:
Helpers use should be minimized. It should not be your go-to place for code 🙈 `bootstrap` should indeed contain all initialization code but that also include any secondary methods like such helpers too. Keeping the implementation code close to where the methods are declared is a best practice. Especially, in this case, the "helper" method does not need to be shared as it will be used only in `bootstrap` and nowhere else.